docs/streams/upgrade-guide.html

```diff
@@ -191,6 +191,17 @@ <h3><a id="streams_api_changes_210" href="#streams_api_changes_210">Streams API
         details, see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-312%3A+Add+Overloaded+StreamsBuilder+Build+Method+to+Accept+java.util.Properties">KIP-312</a>
     </p>
 
+    <p>
+        We are introducing static membership towards KStream users. This feature reduces unnecessary rebalances during normal application upgrades or rolling bounces.
```

Comment:
`KStream` is not the standard term we recommend to call here. We should just use `Kafka Streams` or `Streams API`.